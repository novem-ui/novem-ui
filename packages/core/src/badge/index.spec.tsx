import * as React from 'react'

import { render } from '@testing-library/react'

import Badge from '.'

describe('<Badge />',() => {
  test('should match snapshot', () => {
    const { container } = render(
      <Badge variant="primary">Dummy text</Badge>
    )

    expect(container).toMatchSnapshot();
  })
})

