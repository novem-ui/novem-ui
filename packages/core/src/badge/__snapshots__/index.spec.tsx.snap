// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Badge /> should match snapshot 1`] = `
<div>
  <div
    class="primary css-g1yzty"
  >
    Dummy text
  </div>
</div>
`;
